imports/plugins/core/orders/client/components/orderTable.js

```diff
@@ -34,6 +34,32 @@ const classNames = {
   }
 };
 
+/**
+ * getBillingInfo
+ *
+ * @summary get proper billing object as per current active shop
+ * @param {Object} order - order object to check against
+ * @return {Object} proper billing object to use
+ */
+function getBillingInfo(order) {
```

Comment:
Not sure what the best practices are for React around condensing helper functions into a lib file or how that should be done, but since this is exactly the same as the other getBillingInfo (and getShippingInfo below is also identical to the other file), perhaps it could exist in one place and get imported? @jshimko any best practice thoughts on this?